docs/examples/canary-rollout-use-case/rollout-v2.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: book-info
+  namespace: default
+spec:
+  components:
+    - name: reviews
+      type: webservice
+      properties:
+        image: docker.io/istio/examples-bookinfo-reviews-v2:1.16.2
+        port: 9080
+        volumes:
```

Comment:
What are the volumes used for?

Reply:
storage the log, the container will `crash` if without this volume